packages/shared/src/common/normalize-zip.ts

```diff
@@ -4,8 +4,17 @@
  * @returns The normalized zip code as a string.
  */
 export function normalizeZipCode(zipCode: string): string {
-  if (zipCode.includes("-") && zipCode.length === 9) {
-    return zipCode.slice(0, 4);
+  if (!isValidZipCode(zipCode)) {
+    throw new Error("Zip codes may only contain numbers (0-9) and a dash (-)");
   }
```

Comment:
Prob something to address some other time, but I'm wondering if "normalize" should only remove invalid stuff and try to get it to a certain pattern, returning undefined if it can't, whereas "validate" would throw an error.

Could add a TODO for next time we touch this maybe, since its NABD for the current usage.

Reply:
Sounds good